nibabel/nifti1.py

```diff
@@ -1331,6 +1347,10 @@ def set_intent(self, code, params=(), name=''):
             defaults to ().  Unspecified parameters are set to 0.0
         name : string
             intent name (description). Defaults to ''
+        allow_unknown : bool
```

Comment:
Should be `allow_unknown : {False, True}, optional` to signal default (first value in curlies) and the fact that it's optional.